cluster/vagrant/config-default.sh

```diff
@@ -21,14 +21,14 @@ NUM_MINIONS=${NUM_MINIONS-"1"}
 export NUM_MINIONS
 
 # The IP of the master
-export MASTER_IP="10.245.1.2"
-export KUBE_MASTER_IP="10.245.1.2"
+export MASTER_IP=${MASTER_IP-"10.245.1.2"}
+export KUBE_MASTER_IP=$MASTER_IP
```

Comment:
It's pedantic, but `s/$MASTER_IP/${MASTER_IP}/`. The dominant bash style I've been trying to enforce, though it's waivering, is for `${TOKEN}`, regardless of context.